backend/src/main/java/saci/application/model/Answer.java

```diff
@@ -0,0 +1,22 @@
+package saci.application.model;
+
+public class Answer {
+    private Integer knowledgeId;
+    private Integer answer;
+
+    public Integer getKnowledgeId() {
```

Comment:
Have you checked with @vascolusitan how to generate automatically getters and setters methods, with annotations?